beacon.yaml

```diff
@@ -1705,33 +1751,15 @@ components:
         numTotalResults:
           type: integer
           minimum: 0
-        results:
```

Comment:
If I'm reading this correctly then results will disappear and be replaced by resultSet, shouldn't results still be an array with the items being the resultSets?

Reply:
Good catch! `resultSet` must be an array.

Reply:
Fast :-)

Reply:
Fixed :)